README.md

```diff
@@ -28,3 +49,25 @@ import sys
 cmd = [sys.executable,  '-m', 'pip', 'install', '--user', 'pipster']
 subprocess.check_call(cmd)
 ```
+
+## Usage
+
+The `install` function can be called in two ways.
+
+#### `install('pip install useful_package')`
+
+If you give `install()` a single string that begins with `pip install`, then it
+will run that exact command as if it were given at the command line.
+
+#### `install('useful_package1', 'useful_package2', user=True, ...)`
+
+You can also pass any number of target packages along with commandline
+arguments or keyword arguments corresponding to commandline parameters for `pip
+install`. For example, all of the following lines have the same result:
+
+```python
+$ pip install -r requirements.txt
```

Comment:
@reynoldsnlp I was skimming through the repo and thought I'd post a bit of an educational feedbac here..

Looks like a better lexer choice for this snippet is `pycon` which is intended to be used for showing Python REPL excerpts:
````suggestion
```pycon
# $ pip install -r requirements.txt
````

OTOH, there's a confusion with the shell session mixed in.
When showing snippets of from shell, it's recommended to use `console` instead. The idea is that the lexer would know to highlight shell syntax only on the line that has the shell prompt at the beginning, assuming that the non-prompt bits just represent text output of the commands.
For example, this is a shell session:
```console
$ a-command $(subcommand thing)
some output here $(this is not highlighted)
$ another-command-without-output
```

With the REPL, it'd be something like this:
```pycon
>>> print('text')
text
```

There's also `python-traceback` which can be used for highlighting tracebacks specifically. All of the above can't be mixed in the same snippet. Doing so will leave some parts highlighted incorrectly.